android/src/main/java/com/amplitude/android/internal/locators/ComposeViewTargetLocator.java

```diff
@@ -81,12 +81,19 @@ public ViewTarget locate(
                                 semanticsModifierCore.getSemanticsConfiguration();
                         for (Map.Entry<? extends SemanticsPropertyKey<?>, ?> entry : semanticsConfiguration) {
                             final @Nullable String key = entry.getKey().getName();
-                            if ("OnClick".equals(key)) {
-                                isClickable = true;
-                            } else if ("TestTag".equals(key)) {
-                                if (entry.getValue() instanceof String) {
-                                    lastKnownTag = (String) entry.getValue();
-                                }
+                            switch (key) {
+                                case "ScrollBy":
+                                    isScrollable = true;
+                                    break;
+                                case "OnClick":
+                                    isClickable = true;
+                                    break;
+                                case "SentryTag":
```

Comment:
```suggestion
```

Reply:
Nice catch! Fixed in https://github.com/amplitude/Amplitude-Kotlin/pull/201/commits/c303099245c55d870cae86d3e81136e4dd7d6cc3.